promptsource/templates/enriched_web_nlg/en/templates.yaml

```diff
@@ -0,0 +1,56 @@
+dataset: enriched_web_nlg
+subset: en
+templates:
+  3860d7fb-0b50-4275-a7ab-782ae86756e5: !Template
+    answer_choices: null
+    id: 3860d7fb-0b50-4275-a7ab-782ae86756e5
+    jinja: 'Take the following triple set as part of a Data-to-Text task: {{modified_triple_sets.mtriple_set[0]
+      | join(", ")}}. Make a lexicalization of the triple set into plain text. |||
+      {{lex.text[0]}}'
+    metadata: !TemplateMetadata
+      choices_in_prompt: false
+      metrics:
+      - BLEU
```

Comment:
```suggestion
      - BLEU
      - ROUGE
```

Reply:
ROUGE is not usually reported for Data-to-Text, usually METEOR, TER, but do see it in the E2E paper 👍 so think a good change.

Reply:
Would be good to have multi-reference BLEU to use the references.